basescript/stats.py

```diff
@@ -85,13 +91,16 @@ def make_series_key(measurement, **tags):
         """
         # TODO include retention policy in the series key creation ?
         if len(tags) == 0:
-            return measurement
+            return CLEAN_STR(measurement)
 
-        tags_srtd = sorted([ '%s=%s' % (k,v) for k,v in tags.iteritems() if v is not None ])
+        tags_srtd = sorted([
```

Comment:
@algrebe Some examples of this transformation as comments (or even better doctests) would be helpful to someone reading the code.